lib/drivers/abbott/.eslintrc

```diff
@@ -11,7 +11,8 @@
       {
         "allowForLoopAfterthoughts": true
       }
-    ]
+    ],
+    "lodash/prefer-lodash-method": "off"
```

Comment:
When Dorian developed the Libre driver, he used ES6 methods instead of Lodash methods. Since all the ES6-ified files in that folder is for the Libre, I thought making it generally true for that folder is fine. To be honest, I wouldn't mind making this generally true for all our ES6-ified drivers - I don't quite see why we want to prefer Lodash methods, given that it's an Electron app and we know the ES6 methods will always be available?

Reply:
No problem. I was just checking. I’d been following a previous convention I’d found in the code to prefer lodash. 
I’m happy to make ES6 the new convention :)